evm-contracts/test/v0.6/AccessControlledAggregator.test.ts

```diff
@@ -275,15 +163,15 @@ describe('AccessControlledAggregator', () => {
       describe('with access', () => {
         it('succeeds', async () => {
           await aggregator.connect(personas.Carol).addAccess(testHelper.address)
-          await testHelper.readLatestAnswer(aggregator.address)
+          await testHelper.readGetRoundData(aggregator.address, nextRound)
         })
       })
     })
 
     describe('when read by a regular account', () => {
       describe('without explicit access', () => {
         it('succeeds', async () => {
-          await aggregator.connect(personas.Eddy).latestAnswer()
+          await aggregator.connect(personas.Eddy).getRoundData(nextRound)
```

Comment:
Eddy's new to me.  What's his role?

Reply:
Pretty sure Eddy has been used simply as a second Carol.